packages/serve/startDevServer.ts

```diff
@@ -0,0 +1,23 @@
+import * as Server from "webpack-dev-server/lib/Server";
```

Comment:
/cc @evilebottnawi I made some progress here. In here I put basically the ideal version of what we want, so minimal options handling before starting the server. 

I think `flags.js` should live in `webpack-dev-server` in the future, rather than here, because then we can easily make updates without changing `webpack-cli`, but we can put that in `webpack-dev-server` `next` later. Do you agree?

Reply:
The tests for options should live inside `webpack-cli` - since it owns the corresponding API for said options. 

`webpack-dev-server` should have tests that validates integration & compatibility with `webpack-cli` api. For ex, this would apply to your changes for `webpack-cli/lib/webpack-cli.js` where you have added the `processArgs` fn, etc.